src/components/Calendar/Calendar.jsx

```diff
@@ -48,6 +48,12 @@ const getTop = ({location}) => {
   return 0;
 }
 
+export const calendarFetch = () => Get(new Url(CALENDAR_URL).path("events").queryStrings({
```

Comment:
Thinking about moving these functions to a separate file. Thoughts?

Reply:
I think it could make testing/file exports cleaner as well. Would also set a good precedent if this codebase will be growing in the upcoming semester.